.travis.yml

```diff
@@ -6,6 +6,8 @@ dist: trusty
 env:
   global:
   - TRAVIS_SBT_VERSION="1.2.8"
+  - secure: M3HqXGpiwZ2F9/m9zw3FrkJxRin/Ra4gfBzy39sZV1ZDc9vkxZY5Wf1mkgGyBDXJECr1kzutFkqSANqZK1wQfHnLQFGR0xEnP8XaVGwKMF0QBNJPM5UAryJAP+61l4ASvXj48fyQPzKLSsgzJ0aQDgOQXK2lejHQpcbz7pEJJ/af6b6ocLHwC9lQKyx+iAsdanaUkL40fftwB4GKHPIv0C8BB8qJWSIJl/q/XK9HtAqch7GxOwzesUujZecBMEMvvVw80qU5EIKKaykMNjivDkzjthkgZMjN9zSNUx1p0bi5k3Q3usqAs4wkvbI28s2mbVUvZenxuhIISvTnMmZUjBqW+kuh/5vJLbKkNxrJl5idDe7Aq3LQbS1TunAstfvb/VMj/CJEqBFce1fK6cvlG/l7QlLU5Cb8M1Ze5hTfv+wiEjVbapdb8b9xX34B1B+YCH3cwSN8kyHhaCPTmHGzPfIso/YdnlFvjApAqYNzqoVSex2TB88oKi3SkXchOM41weJXvD+LGqZENT+HAsFwyOyXvHN/Yu3VgMH+d7H8YnDOx/HcRTKg5jm6OaS74Ct3CHe1Wh5TMZ+rj5h9rVJO1uBuzVg6nCF01PsuetGkjv2ib+DmcBA5pcwXF0ImFonLbUN/GD7zeXUdSMDLBGth/6wPVdu7khjDt5gXOTLuxFw=
+  - secure: r3gBqjdG8/TTApJJyIiq+shx2u+pqDLsT5gkJR2Uq3vbNC3LY9aBs63ilEx5Q7qV6RN45AifKIgeynwXQl7K9zImW4yx7hPxyl8HVlZW7fOONjEQM0NWLytNihPJVHqKH71NUHzubG4e/16avRkKG6EyEezKfsTznMtC0nzV1e6qa+kMC1L6DW4POvCntpDPOwVprOoB+m0DFNCAfFmPZETs7K0BnbuayISro87I/58SMplWFZX5aACGHTFvd4SoZUkpJit3osrlDmtQqCT6LrPzKveThmeKWhtkbR0WvdHgRYZtVUeuGdp47tVJi287gtbGBxY7U0oVSYiNM3CIA5QdAbKNBedgSpBZKrmVal9LN19+oPjT+Yn3WtE1x/gPcI/losEpBe94uvEjC+hZaX7KVhcjmDfwyi39YYVL1sT6hoC1kTBICXKbP/V+yQ/Bk04BwX0VPmSvExvMkR6QRGrECyNytoGfWIQsI6pVMAnl21OX6FWxDsK0F5F0jePaM7/zpGtLCuxhmPwp9PkFM4RtY017EZYS2JNlHYfOWEN8ZYW8/CZIfcxRGv5QSGL4mHOlaZ7P2pZmsI7wYuaE7VRheWBW3S+Qp7QfGDGXF5gmS+BNaDJ3/R5TGTdTK9OFH5e8bF2S8WGsDJOaeJdVDFJ+t3FWt76R2+B2SbFI5c8=
```

Comment:
Is it ok that there are 2 secure keys?

Reply:
Those are env vars for publishing to expo and encrypted using travis cli, so no one can use them since they are per repo.

You can find more info here: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

Reply:
Thanks!